### ISSUE TITLE
Upgrade gcloud to google-cloud

### ISSUE THREAD

**Issue by loint** [elegantmonkeys/socket.io-pubsub#1]
Hi there,

Google Cloud has already removed the option `reuseExisting` in their repo so do you have any plan to upgrade it ?

[Google Cloud - Issue 1257](https://github.com/GoogleCloudPlatform/google-cloud-node/issues/1257)

[https://github.com/elegantmonkeys/socket.io-pubsub/blob/master/lib/index.js#L131](https://github.com/elegantmonkeys/socket.io-pubsub/blob/master/lib/index.js#L131)

I saw the recommendation when installing this repo.
```
npm WARN deprecated gcloud@0.37.0: gcloud has been renamed to google-cloud. To get new features and bug fixes, you must use the new package.
npm WARN deprecated node-uuid@1.4.7: use uuid module instead
```

This repo is great and I'm using it in production but do not know the right way to fix. I've already tried many times but un-successful, please help !

Thanks in advance !

**Comment by idoshamun** [comment]
Thanks for the appreciation!

I will do my best to fix it as soon as possible

**Comment by loint** [comment]
Thanks for your great contribution

**Comment by idoshamun** [comment]
Released and published in v1.0.2.
Let me know how it goes :)

**Comment by loint** [comment]
Thanks @idoshamun
